spec/lib/fastly_nsq/fake_message_queue_spec.rb

```diff
@@ -10,6 +10,23 @@
     end
   end
 
+  describe '@@logger' do
+    after do
+      FakeMessageQueue.logger = Logger.new(nil)
```

Comment:
Crazy idea: What about a `FakeMessageQueue.reset_logger!` method which does this?